chingu-coders/Voyage2-Turtles-02
# Browser Updates

## Purpose
* Update both Chrome and Firefox with latest release

## Approach/Fixes
* Pomodoro timer should persist when DevTab is out of focus
* Pomodoro settings are not customizable on input click instead of just the slider
* Sticky notes have padding again
* Updated sticky note colors
* Updated styling on buttons to be more consistent
--------------------------------------------------------------
* Merge all code into development for final testing
* Make another PR afterward to merge into `master` branch

## Testing
* Check to make sure pomodoro timer persists when DevTab is out of focus
* Check the timer setting to see if you can change the input when clicking on the number instead of the slider
* Check that the sticky notes now have padding
* Check the updates settings to see that v1.1.4 is there with the latest changes
* Be sure to check for all of this both on chrome AND firefox
